pysat/instruments/cosmic_gps.py

```diff
@@ -132,8 +132,9 @@ def list_files(tag=None, sat_id=None, data_path=None, format_str=None):
         minute = np.array(stored['minute'])
         try:
             uts = hour*3600.0 + minute*60.0
-        except (TypeError, UFuncTypeError) as err:
-            raise TypeError('unable to construct time from filename\n{:}'.format(err))
+        except TypeError as err:
+            raise TypeError(' '.join(('unable to construct time from',
+                                      'filename\n{:}'.format(str(err)))))
```

Comment:
This change can cause problems.  Do we know that the `UFuncTypeError` won't be raised?

Reply:
`UFuncTypeError` will be caught by `TypeError`.  Checked into the rationale for this in #442.  `UFuncTypeError` is raised when we try to multiply a numpy array of strings by a number.  Test code below.

```
import numpy as np
hour = np.array([20, 21, 22])  
minute = np.array(['1', '2', '3']) 
try: 
    uts = hour*3600.0 + minute*60.0 
except TypeError: 
    print('Everything is fine') 
```